syntax = "proto3";

message PostUnliked {
  string event_type = 1;
  int32 post_id = 2;
}


